plugins/ern_v0.45.0+/react-native_v0.61.0+/Podfile

```diff
@@ -0,0 +1,41 @@
+platform :ios, '{{{iosDeploymentTarget}}}'
```

Comment:
Question for the iOS experts (@jw2175 / @liulianci) - Is `iosDeploymentTarget` the correct terminology here? Or should it just be `iosPlatform` or something else? I've seen slightly inconsistent usage with these terms in the past.

Reply:
I'm not an iOS expert but I personally favor "deployment target", it's clear as to what it maps to.
In XCode project configuration files, this setting is named `IPHONEOS_DEPLOYMENT_TARGET` and in XCode UI it can be set via `iOS Deployment target`. 
`iOS Platform` does not appear anywhere in XCode / project files to refer to the deployment target, so I favor calling a cat a cat ;P

Reply:
Yes makes sense. I was also thinking about the `IPHONEOS_DEPLOYMENT_TARGET ` build config setting.. Somewhat confusing that it's declared in multiple places with slightly different terminology. We'll keep it as `iosDeploymentTarget` then (I assume it is clear enough that at this point this is _only_ for the number in Podfile, _not_ in build config..). Related to that, I wonder if we should make sure those numbers are always aligned if we implement this variable deployment target support.

Reply:
OK I saw you already commented on the alignement in https://github.com/electrode-io/electrode-native/pull/1737 😄 👍